src/components/NewDraftWizard/WizardComponents/WizardFooter.js

```diff
@@ -112,8 +82,10 @@ const WizardFooter = () => {
               color="primary"
               disabled={wizardStep < 1}
               className={classes.footerButton}
+              // TODO: Implement save functionality
```

Comment:
I think we can create an issue out of it for better tracking if there isn't one yet.

Reply:
> NOTE: There needs to be a check if form / upload field values have changed from initial render.

This is addressed in #77 

> I think we can create an issue out of it for better tracking if there isn't one yet.

Yeah, this is a different issue since save functionality is out of scope of dialog alerts.